packages/client/src/i18n/messages/buttons.ts

```diff
@@ -152,7 +152,7 @@ const messagesToDefine: IButtonsMessages = {
     id: 'buttons.forgottenPIN'
   },
   goToHomepage: {
-    defaultMessage: 'Go to Homepage',
+    defaultMessage: 'Back to home',
```

Comment:
Hey! Could you also make these same copy changes to opencrvs-farajaland repo? These default messages are basically only fallbacks for when the actual copy from country config cannot be loaded or is still loading.

You should be able to find the texts here https://github.com/opencrvs/opencrvs-farajaland/blob/f2c83d60b1a5bab2b0d3a822d720446c3f0c734c/src/farajaland/features/languages/generated/client/client.json#L27

Reply:
done!